src/main/java/fr/brouillard/oss/cssfx/CSSFX.java

```diff
@@ -199,11 +224,14 @@ public CSSFXConfig addConverter(URIToPathConverter converter) {
          * @return a Runnable object to stop CSSFX monitoring
          */
         public Runnable start() {
+
             if(Boolean.getBoolean("cssfx.disable")) {
                 System.out.println("CSSFX was not started, because it's disabled via the system property 'cssfx.disable'");
                 return () -> {};
             }
 
+            cssFXStarted.set(true);
+
```

Comment:
we lack a cleanup of this state change during stop action

Reply:
The stop mechanism doesn't really work. 
The listener still stays active. 
And it's not really used in any application I know of.
So I would rather suggest removing the stop method.